response_operations_ui/templates/social-view-case-details.html

```diff
@@ -3,7 +3,7 @@
 {% block main %}
 
 <div class="container page__container">
-    <h1 class="jupiter u-mt-m" id="case_ref">{{attributes.Reference}}</h1>
+    <h1 class="jupiter u-mt-m" id="case_ref">{{attributes.TLA}}{{attributes.REFERENCE}}</h1>
```

Comment:
Might be worth passing the full reference ("LMS0001", for example) as a template variable rather than relying on the attributes.